src/components/apps/editor/ParamGroups.js

```diff
@@ -0,0 +1,235 @@
+/**
+ * A form component for creating, editing, and deleting App groups.
+ *
+ * @author psarando
+ */
+import React from "react";
+
+import { FieldArray, getIn } from "formik";
+import { Trans } from "react-i18next";
+
+import { useTranslation } from "i18n";
+
+import ids from "./ids";
+import styles from "./styles";
+
+import GroupPropertyForm from "./GroupPropertyForm";
+import Parameters from "./Parameters";
+import ParamLayoutActions from "./ParamLayoutActions";
+import ParamPropertyForm from "./ParamPropertyForm";
+
+import ConfirmationDialog from "components/utils/ConfirmationDialog";
+
+import { build as buildID } from "@cyverse-de/ui-lib";
+
+import {
+    Accordion,
+    AccordionSummary,
+    AccordionDetails,
+    Button,
+    Card,
+    CardActions,
+    CardContent,
+    Typography,
+    makeStyles,
+    useTheme,
+    useMediaQuery,
+} from "@material-ui/core";
+
+import { Add, ExpandMore } from "@material-ui/icons";
+
+const useStyles = makeStyles(styles);
+
+function ParamGroupForm(props) {
+    const {
+        baseId,
+        fieldName,
+        group,
+        onDelete,
+        onEdit,
+        onMoveDown,
+        onMoveUp,
+        onEditParam,
+        keyCount,
+        setKeyCount,
+        scrollToField,
+        setScrollToField,
+    } = props;
+
+    const groupEl = React.useRef();
+    React.useEffect(() => {
+        if (groupEl && scrollToField === fieldName) {
+            groupEl.current.scrollIntoView();
+            setScrollToField(null);
+        }
+    }, [fieldName, groupEl, scrollToField, setScrollToField]);
+
+    const theme = useTheme();
+    const isMobile = useMediaQuery(theme.breakpoints.down("xs"));
+    const classes = useStyles();
+
+    const groupBaseId = buildID(baseId, fieldName);
+
+    return (
+        <Accordion ref={groupEl} defaultExpanded>
+            <AccordionSummary
+                className={classes.paramsViewSummary}
+                expandIcon={
+                    <ExpandMore className={classes.paramsViewsExpandIcon} />
+                }
+            >
+                <Typography className={classes.flex} variant="subtitle2">
+                    {group.label}
+                </Typography>
+                <ParamLayoutActions
+                    baseId={groupBaseId}
+                    ButtonProps={{
+                        color: "primary",
+                        variant: isMobile ? undefined : "contained",
```

Comment:
I thought this was required before I did some refactoring, since `variant="contained"` was getting passed to the DOM element of the dot-menu button; but it's not happening anymore, so I can just remove this `isMobile` check now.